### ISSUE TITLE
Add instance for `text-containers

### ISSUE THREAD

**Issue by ChShersh** [kowainik/containers-backpack#17]
This package:

* https://hackage.haskell.org/package/text-containers-0.1.0.0

**Comment by kahlil29** [comment]
I'd like to pick this up. Is there any other previously added package that has been added to this repo that I can use as reference?

**Comment by ChShersh** [comment]
@kahlil29 Yes, the implementation should look pretty much like for `containers-int-strict`. I think we settled with the implementation of the `Map` signature for `IntMap` so the implementation can be take as the example.
